bring to master (#2581)
grafana/loki: docs/clients/fluentd/README.md

> ## Docker Image

Reviewer: ```suggestion
## Docker image
```

> There is a Docker image `grafana/fluent-plugin-loki:master` which contains [default configuration files](https://github.com/grafana/loki/tree/master/fluentd/fluent-plugin-loki/docker/conf). By default, fluentd containers use the configurations but you can also specify your `fluentd.conf` with `FLUENTD_CONF` environment variable.

Reviewer: Is Fluentd a proper name? It is lowercase here and capitalized below. Please capitalize consistently.

This image also uses `LOKI_URL`, `LOKI_USERNAME`, and `LOKI_PASSWORD` environment variables to specify the Loki's endpoint, user, and password (you can leave the USERNAME and PASSWORD blank if they're not used).

> This image will start an instance of Fluentd to forward incoming logs to the specified Loki url. As an alternate, containerized applications can also use [docker driver plugin](../docker-driver/README.md) to ship logs without needing Fluentd.

Reviewer: ```suggestion
This image will start an instance of Fluentd to forward incoming logs to the specified Loki URL. Applications run in containers can also use [Docker driver plugin](../docker-driver/README.md) to ship logs without needing Fluentd.
```
